solv/tdnfpackage.c

```diff
@@ -1683,10 +1681,6 @@ SolvReportProblems(
     {
         fprintf(stderr, "Found %u problem(s) while resolving\n", total_prblms);
     }
-    else
-    {
-        printf("No problems found while resolving\n");
```

Comment:
This is changing the observable behaviour of TDNF in general. Can you please make sure nothing is depending on this text output? for instance, out internal test suite does a lot of checks based on the output strings.